main.c

```diff
@@ -1761,6 +1766,7 @@ int main(int argc, char *argv[])
 		fail(etoplvl, "could not create xdg_toplevel");
 	xdg_toplevel_add_listener(term.toplvl, &toplvl_listener, NULL);
 	xdg_toplevel_set_title(term.toplvl, "havoc");
+	xdg_toplevel_set_app_id(term.toplvl, term.opt.app_id ? term.opt.app_id : "havoc");
```

Comment:
The default should be set here at the top so it's all in one place:
https://github.com/ii8/havoc/blob/7c4614069bbbb069fbf94726b4864d1857cc397a/main.c#L147-L152